pandas/tests/frame/methods/test_append.py

```diff
@@ -167,6 +167,35 @@ def test_append_dtypes(self):
         expected = DataFrame({"bar": Series([Timestamp("20130101"), 1])})
         tm.assert_frame_equal(result, expected)
 
+        df = pd.DataFrame([1], dtype=pd.Int64Dtype())
```

Comment:
can you parametrize these, possibly you will need to split test_append to multiple functions to make this possible

Reply:
@jreback sure, done